2024-10-02 00:54:18,640 DEBUG    root            Loaded Command Group: ['gcloud', 'compute']
2024-10-02 00:54:18,727 DEBUG    root            Loaded Command Group: ['gcloud', 'compute', 'ssh']
2024-10-02 00:54:18,728 DEBUG    root            Running [gcloud.compute.ssh] with arguments: [--project: "c3labs", --zone: "europe-north1-b", [USER@]INSTANCE: "instance-20241001-232901"]
2024-10-02 00:54:18,801 WARNING  root            The private SSH key file for gcloud does not exist.
2024-10-02 00:54:18,801 WARNING  root            The public SSH key file for gcloud does not exist.
2024-10-02 00:54:18,802 WARNING  root            You do not have an SSH key for gcloud.
2024-10-02 00:54:18,802 WARNING  root            SSH keygen will be executed to generate a key.
2024-10-02 00:54:18,802 DEBUG    root            Running command [/usr/bin/ssh-keygen -t rsa -f /Users/MiguelTavares/.ssh/google_compute_engine].
2024-10-02 00:54:18,802 DEBUG    root            Executing command: ['/usr/bin/ssh-keygen', '-t', 'rsa', '-f', '/Users/MiguelTavares/.ssh/google_compute_engine']
2024-10-02 00:54:36,608 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): compute.googleapis.com:443
2024-10-02 00:54:37,025 DEBUG    urllib3.connectionpool https://compute.googleapis.com:443 "GET /compute/v1/projects/c3labs/zones/europe-north1-b/instances/instance-20241001-232901?alt=json HTTP/1.1" 200 None
2024-10-02 00:54:37,036 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): compute.googleapis.com:443
2024-10-02 00:54:37,482 DEBUG    urllib3.connectionpool https://compute.googleapis.com:443 "GET /compute/v1/projects/c3labs?alt=json HTTP/1.1" 200 None
2024-10-02 00:54:37,488 DEBUG    root            OS Login Enabled: False
2FA Enabled: False
Security Keys Enabled: False
Username: MiguelTavares
Third Party User: False
SSH Security Key Support: None
Environment: None
Security Keys:

Signed SSH Key: False
Require Certificates: False

2024-10-02 00:54:37,488 DEBUG    root            SSH Known Hosts File [/Users/MiguelTavares/.ssh/google_compute_known_hosts] could not be opened: Unable to read file [/Users/MiguelTavares/.ssh/google_compute_known_hosts]: [Errno 2] No such file or directory: '/Users/MiguelTavares/.ssh/google_compute_known_hosts'
2024-10-02 00:54:37,488 DEBUG    root            Current SSH keys in project: []
2024-10-02 00:54:37,489 INFO     ___FILE_ONLY___ Updating project ssh metadata...
2024-10-02 00:54:37,490 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): compute.googleapis.com:443
2024-10-02 00:54:38,607 DEBUG    urllib3.connectionpool https://compute.googleapis.com:443 "POST /compute/v1/projects/c3labs/setCommonInstanceMetadata?alt=json HTTP/1.1" 200 None
2024-10-02 00:54:38,611 DEBUG    root            Operations to inspect: [(<Operation
 id: 550457026225976657
 insertTime: '2024-10-01T16:54:38.428-07:00'
 kind: 'compute#operation'
 name: 'operation-1727826878164-623730b4e6d62-188bbbc0-4c002277'
 operationType: 'compute.projects.setCommonInstanceMetadata'
 progress: 0
 selfLink: 'https://www.googleapis.com/compute/v1/projects/c3labs/global/operations/operation-1727826878164-623730b4e6d62-188bbbc0-4c002277'
 setCommonInstanceMetadataOperationMetadata: <SetCommonInstanceMetadataOperationMetadata
 clientOperationId: 'operation-1727826878164-623730b4e6d62-188bbbc0-4c002277'>
 startTime: '2024-10-01T16:54:38.432-07:00'
 status: StatusValueValuesEnum(RUNNING, 2)
 targetId: 514949411544
 targetLink: 'https://www.googleapis.com/compute/v1/projects/c3labs'
 user: 'migtavares6@gmail.com'
 warnings: []>, 3)]
2024-10-02 00:54:38,613 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): compute.googleapis.com:443
2024-10-02 00:54:47,516 DEBUG    urllib3.connectionpool https://compute.googleapis.com:443 "POST /compute/v1/projects/c3labs/global/operations/operation-1727826878164-623730b4e6d62-188bbbc0-4c002277/wait?alt=json HTTP/1.1" 200 None
2024-10-02 00:54:47,519 DEBUG    root            Operations to inspect: [(<Operation
 endTime: '2024-10-01T16:54:47.306-07:00'
 id: 550457026225976657
 insertTime: '2024-10-01T16:54:38.428-07:00'
 kind: 'compute#operation'
 name: 'operation-1727826878164-623730b4e6d62-188bbbc0-4c002277'
 operationType: 'compute.projects.setCommonInstanceMetadata'
 progress: 100
 selfLink: 'https://www.googleapis.com/compute/v1/projects/c3labs/global/operations/operation-1727826878164-623730b4e6d62-188bbbc0-4c002277'
 setCommonInstanceMetadataOperationMetadata: <SetCommonInstanceMetadataOperationMetadata
 clientOperationId: 'operation-1727826878164-623730b4e6d62-188bbbc0-4c002277'
 perLocationOperations: <PerLocationOperationsValue
 additionalProperties: [<AdditionalProperty
 key: 'europe-north1-a'
 value: <SetCommonInstanceMetadataOperationMetadataPerLocationOperationInfo
 state: StateValueValuesEnum(DONE, 1)>>, <AdditionalProperty
 key: 'europe-north1-b'
 value: <SetCommonInstanceMetadataOperationMetadataPerLocationOperationInfo
 state: StateValueValuesEnum(DONE, 1)>>, <AdditionalProperty
 key: 'europe-north1-c'
 value: <SetCommonInstanceMetadataOperationMetadataPerLocationOperationInfo
 state: StateValueValuesEnum(DONE, 1)>>]>>
 startTime: '2024-10-01T16:54:38.432-07:00'
 status: StatusValueValuesEnum(DONE, 0)
 targetId: 514949411544
 targetLink: 'https://www.googleapis.com/compute/v1/projects/c3labs'
 user: 'migtavares6@gmail.com'
 warnings: []>, 3)]
2024-10-02 00:54:47,519 INFO     ___FILE_ONLY___ Updated [https://www.googleapis.com/compute/v1/projects/c3labs].

2024-10-02 00:54:47,520 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): compute.googleapis.com:443
2024-10-02 00:54:47,906 DEBUG    urllib3.connectionpool https://compute.googleapis.com:443 "GET /compute/v1/projects/c3labs?alt=json HTTP/1.1" 200 None
2024-10-02 00:54:48,087 INFO     ___FILE_ONLY___ Waiting for SSH key to propagate.

2024-10-02 00:54:48,088 DEBUG    root            Running command [/usr/bin/ssh -T -i /Users/MiguelTavares/.ssh/google_compute_engine -o CheckHostIP=no -o HashKnownHosts=no -o HostKeyAlias=compute.1849898479687645437 -o IdentitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/Users/MiguelTavares/.ssh/google_compute_known_hosts MiguelTavares@34.88.73.21 -- :].
2024-10-02 00:54:48,088 DEBUG    root            Executing command: ['/usr/bin/ssh', '-T', '-i', '/Users/MiguelTavares/.ssh/google_compute_engine', '-o', 'CheckHostIP=no', '-o', 'HashKnownHosts=no', '-o', 'HostKeyAlias=compute.1849898479687645437', '-o', 'IdentitiesOnly=yes', '-o', 'StrictHostKeyChecking=no', '-o', 'UserKnownHostsFile=/Users/MiguelTavares/.ssh/google_compute_known_hosts', 'MiguelTavares@34.88.73.21', '--', ':']
2024-10-02 00:55:03,590 DEBUG    root            Running command [/usr/bin/ssh -t -i /Users/MiguelTavares/.ssh/google_compute_engine -o CheckHostIP=no -o HashKnownHosts=no -o HostKeyAlias=compute.1849898479687645437 -o IdentitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/Users/MiguelTavares/.ssh/google_compute_known_hosts MiguelTavares@34.88.73.21].
2024-10-02 00:55:03,591 DEBUG    root            Executing command: ['/usr/bin/ssh', '-t', '-i', '/Users/MiguelTavares/.ssh/google_compute_engine', '-o', 'CheckHostIP=no', '-o', 'HashKnownHosts=no', '-o', 'HostKeyAlias=compute.1849898479687645437', '-o', 'IdentitiesOnly=yes', '-o', 'StrictHostKeyChecking=no', '-o', 'UserKnownHostsFile=/Users/MiguelTavares/.ssh/google_compute_known_hosts', 'MiguelTavares@34.88.73.21']
2024-10-02 09:58:33,834 INFO     ___FILE_ONLY___ 
Recommendation: To check for possible causes of SSH connectivity issues and get
recommendations, rerun the ssh command with the --troubleshoot option.

gcloud compute ssh instance-20241001-232901 --project=c3labs --zone=europe-north1-b --troubleshoot

Or, to investigate an IAP tunneling issue:

gcloud compute ssh instance-20241001-232901 --project=c3labs --zone=europe-north1-b --troubleshoot --tunnel-through-iap


2024-10-02 09:58:33,840 DEBUG    root            (gcloud.compute.ssh) [/usr/bin/ssh] exited with return code [255].
Traceback (most recent call last):
  File "/Users/MiguelTavares/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 998, in Execute
    resources = calliope_command.Run(cli=self, args=args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/MiguelTavares/google-cloud-sdk/lib/googlecloudsdk/calliope/backend.py", line 842, in Run
    resources = command_instance.Run(args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/MiguelTavares/google-cloud-sdk/lib/surface/compute/ssh.py", line 450, in Run
    raise e
  File "/Users/MiguelTavares/google-cloud-sdk/lib/surface/compute/ssh.py", line 443, in Run
    return_code = cmd.Run(
                  ^^^^^^^^
  File "/Users/MiguelTavares/google-cloud-sdk/lib/googlecloudsdk/command_lib/util/ssh/ssh.py", line 1848, in Run
    raise CommandError(args[0], return_code=status)
googlecloudsdk.command_lib.util.ssh.ssh.CommandError: [/usr/bin/ssh] exited with return code [255].
2024-10-02 09:58:33,849 ERROR    root            (gcloud.compute.ssh) [/usr/bin/ssh] exited with return code [255].
